hoomd/custom_action.py

```diff
@@ -134,3 +134,13 @@ class _InternalCustomAction(CustomAction, _HOOMDGetSetAttrBase):
     be created.
     """
     pass
+
+
+class ParticleDataFlags(IntEnum):
```

Comment:
Can we make this a nested class inside `CustomAction` as it is only used there?
Also, and maybe outside the scope of this PR, can we move `_CustomOperation`, `CustomAction` and `Logger` out of the root namespace add into submodules/packages? I’d like to keep the root namespace clean with only the objects that all HOOMD simulations interact with.